apps/dg/components/case_table/case_table_view.js

```diff
@@ -584,8 +584,8 @@ DG.CaseTableView = SC.View.extend( (function() // closure
           this.invokeNext(function() {
             gridDataView.expandGroup(parentCaseID);
             gridAdapter.refresh();
-          });
-        });
+          }.bind(this));
```

Comment:
Is this interior bind necessary? The function references are all in the closure.

Reply:
Not strictly, but I went with consistency. It looked curious with only one `bind()`.

Reply:
OK, that's fine. I was just wondering whether I missed something.